src/R/Wpf/Impl/Microsoft.R.Wpf.csproj

```diff
@@ -0,0 +1,80 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I want to split R-Specific styles, brushes, converters etc. from generic things like lambda-converter